docs/operator-guide/cd/manual-approval.md

```diff
@@ -0,0 +1,109 @@
+# Manual Approval in Deploy Pipelines
+
+Since the 3.10.0 release, KubeRocketCI features manual approval task in deploy pipelines, which stops the pipeline execution until the user makes a decision or pipeline is timed out. This feature allows to integrate a quality control checkpoint in your pipelines, reducing risks when deploying applications to production environments.
+
+## Manual Approval Structure
+
+A deploy pipeline with a manual approval step is a typical Tekton pipeline but with an [ApprovalTask](https://github.com/epam/edp-tekton/blob/48f0d18ecfec049618999dabd9c8cbc892ae3508/charts/pipelines-library/templates/pipelines/cd/deploy-with-approve.yaml#L61) custom resource included. The custom deploy pipeline usage flow involves the following steps:
+
+```mermaid
+graph LR;
+    A(Create TriggerTemplate resource) --> B(Create Pipeline resource) --> C(Apply changes to environment) --> D(Run deploy pipeline) --> E(Approve/reject deployment)
+```
+
+## Create Pipeline With Approval Task
+
+To create a deploy pipeline with a manual approval task, follow the steps below:
+
+1. Create and apply the **TriggerTemplate** custom resource. Refer to the example [TriggerTemplate](https://github.com/epam/edp-tekton/blob/48f0d18ecfec049618999dabd9c8cbc892ae3508/charts/pipelines-library/templates/triggers/cd/deploy-with-approve.yaml). Make sure to specify the correct pipeline type in the labels and provide the pipeline reference in the specifications:
+
+  ```bash
+  ...
+  labels:
+      app.edp.epam.com/pipelinetype: deploy
+  ...
+  ```
+
+  ```bash
+  ...
+  spec:
+    taskRunTemplate:
+      serviceAccountName: tekton
+    pipelineRef:
+      name: deploy-with-promote-approval
```

Comment:
this is default pipeline? do we need to enable custom name here?

Reply:
Yes, this is a snippet taken from the default pipeline I provided a link to. In my opinion, we don't need to enter a custom name for it.